### PR TITLE
C380736 Search created MARC bib record by Title, OCLC number (spitfire) (TaaS)

### PR DESCRIPTION
**Pull request by elen2345**
[TestRail](https://foliotest.testrail.io/index.php?/cases/view/380736)
[AllureReport](https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/5433/allure/)
image

### CODE REVIEW

File: cypress/support/fragments/inventory/inventorySearchAndFilter.js [hunk 1]
```diff
@@ -99,13 +100,31 @@ const searchHoldingsByHRID = (hrid) => {
...
+const searchInstanceByKeyword = (keyword) => {
+  cy.do([
+    Select({ id: 'input-inventory-search-qindex' }).choose(
+      'Keyword (title, contributor, identifier, HRID, UUID)',
+    ),
+    TextArea({ id: 'input-inventory-search' }).fillIn(keyword),
+    searchButton.click(),
+  ]);
+};
+
```
Comment (Sherzod-Kenjaev) [inline cypress/support/fragments/inventory/inventorySearchAndFilter.js:112]: You can use method "InventoryInstances.searchInstancesWithOption()" instead the one you have created here. Please try to use already existing methods, try to create new method if there is no related method available.

File: cypress/support/fragments/inventory/inventorySearchAndFilter.js [hunk 2]
```diff
@@ -99,13 +100,31 @@ const searchHoldingsByHRID = (hrid) => {
...
 };
 
+const searchInstanceByOCLC = (oclc) => {
+  cy.do([
+    Select({ id: 'input-inventory-search-qindex' }).choose('OCLC number, normalized'),
+    TextArea({ id: 'input-inventory-search' }).fillIn(oclc),
+    searchButton.click(),
+  ]);
+};
+
```
Comment (Sherzod-Kenjaev) [inline cypress/support/fragments/inventory/inventorySearchAndFilter.js:127]: You can use method "InventoryInstances.searchInstancesWithOption()" instead the one you have created here. Please try to use already existing methods, try to create new method if there is no related method available.

**Review by Sherzod-Kenjaev — CHANGES_REQUESTED** [review]

File: cypress/support/fragments/inventory/inventorySearchAndFilter.js (hunk 1 above)
Comment (elen2345) [reply cypress/support/fragments/inventory/inventorySearchAndFilter.js]: Fixed, thank you!

File: cypress/support/fragments/inventory/inventorySearchAndFilter.js (hunk 2 above)
Comment (elen2345) [reply cypress/support/fragments/inventory/inventorySearchAndFilter.js]: Fixed

**Review by Sherzod-Kenjaev — APPROVED** [review]

**Pull request merged**